.gitignore

```diff
@@ -1,2 +1,6 @@
 /target
-/Cargo.lock
```

Comment:
I'm not a big fan of committing Cargo.lock.

It is not recommended in library crates.